release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -471,14 +471,23 @@ OVN-Kubernetes network traffic can be viewed with OVS sampling via the CLI for t
 * `NetworkPolicy`
 * `AdminNetworkPolicy`
 * `BaselineNetworkPolicy`
-* `UserDefinesdNetwork` isolation
+* `UserDefinedNetwork` isolation
 * `EgressFirewall`
 * Multicast ACLs.
 
 Checking OVN-Kubernetes network traffic with OVS sampling using the CLI is intended to help with packet tracing. It can also be used while the Network Observability Operator is installed.
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/ovn-kubernetes-troubleshooting-sources.adoc#nw-ovn-kubernetes-observability_ovn-kubernetes-sources-of-troubleshooting-information[Checking OVN-Kubernetes network traffic with OVS sampling using the CLI].
 
+[id="ocp-release-notes-networking-user-defined-network-segmentation_{context}"]
+==== User-defined network segmentation (Generally Available)
+
+With {product-title} {product-version}, user-defined network segmentation is generally available. User-defined networks (UDN) introduce enhanced network segmentation capabilities by allowing administrators to define custom network topologies using namespace-scoped UserDefinedNetwork and cluster-scoped ClusterUserDefinedNetwork custom resources.
+
+With UDNs, administrators can create tailored network topologies with enhanced isolation, IP address management for workloads, and advanced networking features. Supporting both Layer 2 and Layer 3 topology types, user-defined network segmentation enables a wide range of network architectures and topologies, enhancing network flexibility, security, and performance. For more information on supported features, see xr3f:../../networking/multiple_networks/understanding-multiple-networks.adoc#nw-udn-support-matrix-primary-secondary[UDN support matrix].
+
+UDN's use cases include providing virtual machines (VMs) with a lifetime duration for static IP addresses assignment as well as a Layer 2 primary pod network so that users can live migrate VMs between nodes. These features are all fully equipped in {VirtProductName}. Users can use UDNs to create a stronger, native multi-tenant environment, allowing you to secure your overlay Kubernetes network, which is otherwise open by default. For more information, see xref:../networking/multiple_networks/primary_networks/about-user-defined-networks.adoc#about-user-defined-networks[About user-defined networks].
```

Comment:
Suggestion to consider writing as "Use cases of...." instead of using UDN's

https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives

> Do not use possessive ’s with inanimate objects.